app/concepcts/reservations/representers/single.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Reservations
+  module Representers
+    class Single
+      def initialize(reservation)
+        @reservation = reservation
+      end
+
+      def call
+        {
+          status: reservation.status,
+          paid: reservation.paid,
+          screening: reservation.screening.movie.title,
+          start_time: reservation.screening.start_time,
```

Comment:
Remember that in the future you can reuse existing representers in other representers :)
For now, we can skip it, just have it in your mind.